content/substrate/intro/slides.md

```diff
@@ -103,8 +103,8 @@ Notes:
 
 Bitcoin block size has never been and is an ongoing debate.
 
-I am not against L2s per se, but it is truth that they mostly exist because the underlying protocol
-is too hard/slow to upgrade itself. ETH Gas prices is also shows that the underlying protocol cannot
+I am not against L2s per se, but it is true that they mostly exist because the underlying protocol
+is too hard/slow to upgrade itself. ETH Gas prices also show that the underlying protocol cannot
 meet the demands of today.
```

Comment:
```suggestion
I am not against L2s per se, but it is true that they mostly exist because the underlying protocol is too hard/slow to upgrade itself.
ETH Gas prices also show that the underlying protocol cannot meet the demands of today.
```